frontend/public/components/pod.tsx

```diff
@@ -250,7 +250,7 @@ const PodGraphs = requirePrometheus(({ pod }) => (
       <div className="col-md-12 col-lg-4">
         <Area
           title="CPU Usage"
-          humanize={humanizeCpuCores}
+          humanize={humanizeCpuCoresLong}
```

Comment:
Do we need to change these? If so, we need to fix the overlap in the charts. I'd almost prefer to leave these pages as they are.

Reply:
My understanding from the discussion so far is, that the `m` unit is not recognized by the user and `cores`/`millicores` are expected instead (wherever possible). If this is valid, we should change other places as well.

No doubt, the `millicores` is long and so precious space on the screen will be wasted just to reserve it for single occurrence within multiple columns/rows. That was the reason for former `mcores` which would not need any additional change in spacing.

Reply:
I'm not sure... If you don't know what `m` is you probably don't know what `millicores` are either. At some point if you want to work with these values in k8s, you'll have to learn about CPU units. `millicores` does have the advantage that it's easier to search for.

I think the more confusing one is actually `cores`, which has *no* unit in some contexts.

A tooltip explaining CPU units might be more valuable than spelling it out everywhere. I'd be in favor of leaving these specific charts as they are.

Reply:
How about we use `# m` for CPU chart y-axis labels for now, but make sure to include "millicores" and "cores" in the black tooltip that appears on hover, like Marek's screenshot below. That gives users at least a slightly better chance at learning without googling.

![image](https://user-images.githubusercontent.com/9122899/69198348-69bc2700-0b02-11ea-9cae-ef5fb137d1f5.png)

In the longer term maybe we can improve/redesign the Utilization card to work better at full widths, allowing us to show the words "cores" and "millicores" without hovering and without feeling forced to squeeze them into the y-axis labels.